design/threat-model/threat-model.md

```diff
@@ -124,15 +125,13 @@ Utilizing VFIO, devices can be passed through to the virtual machine. We will as
 host is limited to gaps in device pass-through handling. This is supported in QEMU and Cloud Hypervisor, but not
 Firecracker.
 
-
 #### ACPI
 
-ACPI is necessary for hotplug of CPU, memory and devices. ACPI is available only in QEMU. In Firecracker and 0.3 Cloud Hypervisor,
-there isn’t the capability to hotplug deviecs, memory, or CPUs.
+ACPI is necessary for hotplug of CPU, memory and devices. ACPI is available in QEMU and Cloud Hypervisor. Device, CPU and memory hotplug
+are not available in Firecracker.
 
 ## Devices and threat model
 
 ![Threat model](threat-model-boundaries.svg "threat-model")
 
 ### Risks
```

Comment:
Are you planning to add anything here?

Reply:
Was considering a "tbd" but just dropped it for now.